mimic/util/helper.py

```diff
@@ -15,6 +15,25 @@
 fmt = '%Y-%m-%dT%H:%M:%S.%fZ'
 
 
+def random_string(length, selectable=None):
+    """
+    Create a random string of the specified length.
+
+    :param int length: How long the string must be.
+    :param str selectable: If left unspecified, the random character selection
+        will be taken from uppercase and lowercase letters, digits, and a few
+        punctuation marks.  Otherwise, the characters will be taken from the
+        string provided.
+    :returns: A string of length `length`.
+    """
+    selectable = selectable or (
+        "abcdefghijklmnopqrstuvwxyz"
```

Comment:
Nonblocking - this is totally fine, and in fact may be more clear in some ways, but you can also use `strings.ascii_lowercase + strings.ascii_uppercase` (or just `strings.ascii_letters`), plus `strings.digits` :)

I'm not entirely sure about the symbols - I see that there exists a `strings.punctuation`, but not sure what that means - probably listing all the symbols is a better idea.

Anyway, totally nonblocking.  :)